data.js

```diff
@@ -4285,6 +4285,13 @@ var tabledata = [
  last: "20240123",
  desc: "Scan file with Yara rules."},
 
+{name: "YaraForge",
+ url: "https://github.com/zhaoxinnZ/YaraForge",
+ src: "py",
+ cats: "int",
+ last: "20240531",
```

Comment:
Do you have a latest version of IDA Pro that this plugin is known to work with? (What version was it tested with?)

Reply:
We've tested our plugin from IDA 7.5 to 8.3 and they're fine